application/common/models/User.php

```diff
@@ -343,6 +343,55 @@ private function savePassword()
         return true;
     }
 
+    public static function getExpiringUsers($criteria): array
```

Comment:
We're matching for a specific date, but including the time for clarity to the end user. This is what I expected everywhere for returned expiration and grace times actually.